qutebrowser/browser/downloads.py

```diff
@@ -580,6 +583,9 @@ def on_reply_error(self, code):
         if code == QNetworkReply.OperationCanceledError:
             return
         else:
+            if self.reply is None:
+                log.downloads.debug("QNetworkReply disappeared %s", self)
+                return
```

Comment:
Seems like I can reproduce it, if you remove this check and do

    :open machdochmal.blogspot.de
    :download-whole /dev/null

qute will crash:

    ERROR: Uncaught exception
    Traceback (most recent call last):
      File "/home/daniel/Code/qutebrowser/qutebrowser/browser/downloads.py", line 591, in on_reply_error
        self._die(self.reply.errorString())
    AttributeError: 'NoneType' object has no attribute 'errorString'

On further investigation, seems like the offending URL is `http://pagead2.googlesyndication.com/pagead/js/google_top_exp.js`, which is hostblocked, and `:download http://pagead2.googlesyndication.com/pagead/js/google_top_exp.js` produces `ERROR: Download error: %HOSTBLOCK%`

Now I'm not sure why it behaves that way when used with `:download-whole`...